src/client/components/login-form.js

```diff
@@ -0,0 +1,100 @@
+import React, { Component } from "react";
+import { Route, Redirect } from "react-router-dom";
+
+class LoginForm extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      email: "",
+      password: "",
+      isLoggedin: false
+    };
+  }
+
+  clickHandlerCancel = event => {
+    this.setState({
+      email: "",
+      password: ""
+    });
+  };
+
+  handleChange = event => {
+    this.setState({
+      [event.target.name]: event.target.value
+    });
+  };
+
+  handleSubmit = event => {
+    event.preventDefault();
+
+    try {
+      // TODO: Add environment variable
+      fetch("api/auth/login", {
```

Comment:
maybe services/api-calls should be in a separate utility-like js file? we can talk about this tomorrow. maybe also you should use [axios](https://github.com/axios/axios)?

Reply:
Might be a good idea. `fetch()` some times acts a little strange with error-handling. I just feel like `fetch()` is good to learn as it will probably be the way of the future, as it is a native browser api. But I agree that we would probably replace it with axios eventually.

Reply:
I have replaced axis instead of fetch